tasks/containers/TaskWorker.js

```diff
@@ -79,14 +79,32 @@ export default compose(
 
```

Comment:
@iainkirkpatrick Woops, that preview didn't work. This line https://github.com/root-systems/cobuy/blob/feature/task-completion/tasks/containers/TaskWorker.js#L78

Turns out https://github.com/ahdinosaur/feathers-action/pull/20 Didn't work how I intended anyway so might have to rethink things.